inf-clojure.el

```diff
@@ -649,12 +680,19 @@ If you are using REPL types, it will pickup the most approapriate
   :type 'string
   :package-version '(inf-clojure . "2.0.0"))
 
+(defcustom inf-clojure-completion-form-planck
+  "(doall (map str (planck.repl/get-completions \"%s\")))\n"
```

Comment:
this `(doall (map str ...` was necessary because lumo returns `#js`. Does planck return it as well?